third_party/amd/include/TritonAMDGPUTransforms/Passes.td

```diff
@@ -25,6 +25,9 @@ def TritonAMDGPUStreamPipeline : Pass<"tritonamdgpu-stream-pipeline", "mlir::Mod
     Option<"localPrefetch", "local_prefetch",
            "int32_t", /*default*/"0",
            "Set local prefetch stage count">,
+    Option<"useAsyncCopy", "use_async_copy",
+           "bool", /*default*/"false",
+           "Use AsyncCopyGlobalToLocal to directly load to shared memory (uses more shared memory)">,
```

Comment:
Why "(uses more shared memory)"?

Reply:
I removed that part as other parameters are also not explained in depth here and it documented in the code.